components/utilities/themeToggle.js

```diff
@@ -1,13 +1,30 @@
-import { useState, useEffect } from "react";
+import React, { useState, useEffect } from "react";
 
 const ThemeToggle = () => {
   const [activeTheme, setActiveTheme] = useState(document.body.dataset.theme);
```

Comment:
Rename activeTheme to activeThemeV1

and tailwindTheme to activeThemeV2

Reply:
Fixed on 594a406